Alpaca.Markets/Interfaces/IHistoricalItems.cs

```diff
@@ -36,12 +36,12 @@ public interface IHistoricalItems<out TItem>
         /// Number of aggregates (minutes or days) used to generate the response.
         /// Polygon v2 API only.
         /// </summary>
-        Int64 QueryCount { get; }
+        UInt64 QueryCount { get; }
```

Comment:
Rollback changes in this interface and its implementation - both will be removed from SDK soon.